test/server/meta.test.js

```diff
@@ -218,6 +218,56 @@ test('should unpack a valid sdk meta bundle with multiple components', () => {
     }
 });
 
+test('should unpack a valid sdk meta bundle with multiple merchant-id email addresses', () => {
+    const emails = [
+        'test@gmail.com',
+        'foo@bar.com',
+        'test@test.org.uk',
+        'test-test@test.com',
+        'test.test@test.com',
+        'test@test@test.com'
+    ];
+
+    const sdkUrl = `https://www.paypal.com/sdk/js?client-id=foo&merchant-id=${ emails.map(anEmail => encodeURIComponent(anEmail)).join(',') }`;
+
+    const { getSDKLoader } = unpackSDKMeta(Buffer.from(JSON.stringify({
+        url: sdkUrl
+    })).toString('base64'));
+
+    const $ = cheerio.load(getSDKLoader());
+    const src = $('script').attr('src');
+
+    if (src !== sdkUrl) {
+        throw new Error(`Expected script url to be ${ sdkUrl } - got ${ src }`);
+    }
+});
+
+test('should error out from invalid merchant-id email addresses', () => {
+    const emails = [
+        '@',
+        '@io',
+        '@test.com',
+        `${ 'a-very-long-email'.repeat(15) }@a-very-long-domain.com`
```

Comment:
is there a reason this isn't a separate test? Since long emails throw a different error, I'd expect them to be unit tested separately from these other ones that all test the same path.

Reply:
They were all grouped in the same test because all of them are errors related to email-address. But it is a good idea, I will use another test